### PR TITLE
Fixes #29969 - applied errata host filter

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Adds filter for hosts, although it feels bit hacky.

Needs https://github.com/Katello/katello/pull/8884

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
Katello PR is in, I believe this is ready :)

File: app/views/unattended/report_templates/host_-_applied_errata.erb [hunk 1]
```diff
@@ -46,11 +54,12 @@ require:
   version: 3.12.0
 -%>
 <%- report_headers 'date', 'hostname', 'erratum_id', 'erratum_type', 'status' -%>
-<%- load_errata_applications(filter_errata_type: input('Filter Errata Type'), 
+<%- load_errata_applications(filter_errata_type: input('Filter Errata Type'),
                              include_last_reboot: input('Include Last Reboot'),
                              since: input('Since'),
                              up_to: input('Up to'),
                              status: input('Status')
```
Comment (ares) [inline app/views/unattended/report_templates/host_-_applied_errata.erb]: missing comma causes syntax error

**Review by ares — CHANGES_REQUESTED** [review]
There's a syntax error

File: app/views/unattended/report_templates/host_-_applied_errata.erb (hunk 1 above)
Comment (ezr-ondrej) [reply app/views/unattended/report_templates/host_-_applied_errata.erb]: :facepalm: although here is an answer to "What is RuboCop good for?" ^_^
"Find developer's stupid mistakes before it even gets to the reviewer"

**Comment by ares** [comment]
[test foreman]

File: app/views/unattended/report_templates/host_-_applied_errata.erb [hunk 2]
```diff
@@ -46,11 +54,12 @@ require:
...
 -%>
 <%- report_headers 'date', 'hostname', 'erratum_id', 'erratum_type', 'status' -%>
-<%- load_errata_applications(filter_errata_type: input('Filter Errata Type'), 
+<%- load_errata_applications(filter_errata_type: input('Filter Errata Type'),
                              include_last_reboot: input('Include Last Reboot'),
                              since: input('Since'),
                              up_to: input('Up to'),
-                             status: input('Status')
+                             status: input('Status'),
+                             host_filter: input('Host filter').presence
```
Comment (ares) [inline app/views/unattended/report_templates/host_-_applied_errata.erb]: `Host filter` vs `Hosts filter` causes `ERF01-2862 [Foreman::Renderer::Errors::UndefinedInput]: Rendering failed, no input with name "Host filter" for input macro found` when I try to render the report

**Review by ares — CHANGES_REQUESTED** [review]
name of the input is out of sync, making this not working at all :-)

**Review by ares — DISMISSED** [review]
Works well, ready to merge after jenkins confirms

**Review by ares — APPROVED** [review]
I'm sorry, Jenkins confirmed :-) Thanks @ezr-ondrej, I think you have the first demo item for 2021 :-)

**Pull request merged**